docs/modeling-your-data/running-data-models-via-snowplow-bdp/standard-models/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Standard models"
+sidebar_position: 2
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+### Overview
+Standard data models are authored and maintained by Snowplow. Follow below steps to create one.
+
+### 1. Create a warehouse connection
+Begin by creating a new warehouse connection. It will be used by a data model to connect to your warehouse. To create a new warehouse connection, click on the "Setup new target" button on the Data Models page and fill in all the necessary details.
```

Comment:
```suggestion
Begin by creating a new warehouse connection. It will be used by a data model to connect to your warehouse. Click on the "Setup new target" button on the [Data Models page](https://console.snowplowanalytics.com/data-modelling) and fill in all the necessary details.
```